test/helpers.js

```diff
@@ -31,20 +31,30 @@ exports.compile = function compile(compiler) {
   });
 };
 
-exports.createCompiler = function createCompiler(options) {
-  const compiler = webpack(options || {
-    bail: true,
+exports.createCompiler = function createCompiler(options = {}) {
+  const compiler = webpack({
     cache: false,
     entry: `${__dirname}/stubs/entry.js`,
+    resolve: {
+      unsafeCache: false,
```

Comment:
Probably not needed... I added it to debug an issue while i was making the es test.. Ill go ahead and remove it.